packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -173,14 +173,15 @@ export default function NewCommentModal({
       contentRef={modalRef}
       onRequestClose={confirmClose}
       kind={Modal.Kind.FlexibleCenter}
-      size={Modal.Size.Small}
+      size={Modal.Size.Medium}
       tabs={Object.values(CommentTabs)}
       {...props}
+      className="min-h-[25rem]"
     >
       <Modal.Header.Tabs
         disabledTab={(tab) => tab === CommentTabs.Preview && disabled}
       />
-      <Modal.Body view={CommentTabs.Write}>
+      <Modal.Body className="min-h-[15rem]" view={CommentTabs.Write}>
```

Comment:
Same here, @el what do you think?

Reply:
The `min-h-[25rem]` on the container is the desired output. Kindly advise how you think we should proceed.